Gemfile

```diff
@@ -64,6 +65,7 @@ group :test do
   gem 'codeclimate-test-reporter', '~> 1.0.0'
   gem 'factory_girl_rails'
   gem 'json'
+  gem 'rails-controller-testing'
```

Comment:
For access to `response` and `assigns` – got extracted into a gem with Rails 5.0